src/com/geekhub/hw3/auth/AuthenticationService.java

```diff
@@ -24,7 +27,23 @@ public static void main(String[] args) {
     };
 
     private User auth(String login, String password) throws AuthException {
-        //TODO: Implement me
+        if(login.length()==0 | password.length()==0) {
+            throw new WrongCredentialsException("Please provide a login and a password");
+        }
+        else {
+            for(User usr: users)
```

Comment:
For loop will go throught all users even if needed user is first one in the list